LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2009-2015 Yola, Inc.
```

Comment:
Some people seem to add:

```
The MIT License (MIT)

```

at the top too. So people can figure it out with a glance.